default/python/AI/FreeOrionAI.py

```diff
@@ -102,15 +102,15 @@ def startNewGame(aggression_input=fo.aggression.aggressive):  # pylint: disable=
 def resumeLoadedGame(saved_state_string):  # pylint: disable=invalid-name
     """Called by client to when resume a loaded game."""
     if fo.getEmpire() is None:
-        print "This client has no empire. Doing nothing to resume loaded game."
+        info("This client has no empire. Doing nothing to resume loaded game.")
```

Comment:
As above, this seems like enough of an unexpected situation that it should be debug or warn rather than merely info.